amm/runtime/src/main/scala/ammonite/runtime/Storage.scala

```diff
@@ -188,23 +188,23 @@ object Storage{
     // on every version bump; otherwise binary-incompatible changes to
     // ReplAPI/Preprocessor/ammonite-ops will cause scripts to fail after
     // someone upgrades Ammonite.
-    val cacheDir = dir/'cache/ammonite.Constants.version
-    val compileCacheDir = cacheDir/'compile
+    val cacheDir = dir/"cache"/ammonite.Constants.version
+    val compileCacheDir = cacheDir/"compile"
     val classFilesOrder = "classFilesOrder.json"
     val ivyCacheFile = cacheDir/"ivycache.json"
     val coursierFetchCacheDir = cacheDir/"coursier-fetch-cache"
     val metadataFile = "metadata.json"
     val sessionFile  = dir/"session"
 
-    def getSessionId() = {
+    def getSessionId: Long = {
```

Comment:
The now removed `()` was probably there to indicate the side effect of reading the file. I'm neutral to this change, though.

Reply:
This method overrides an abstract method that is defined without the `()` argument lists.
In Scala 3 overrides need to have the same signature, so I preferred to maintain the interface one.